projects/zetasql/build.sh

```diff
@@ -18,6 +18,8 @@
 # This build.sh is partly modeled after that of envoyproxy:
 # https://github.com/google/oss-fuzz/blob/master/projects/envoy/build.sh
 
+set -e
```

Comment:
is this really needed? I think it's enabled on the first line already

Reply:
Yes you're right. I didn't pay attention when I added this line.